server/src/main/java/io/spine/server/aggregate/AggregateStorage.java

```diff
@@ -84,9 +85,11 @@ protected void checkNotClosed() throws IllegalStateException {
     /**
      * Writes events into the storage.
      *
-     * <p>NOTE: does not rewrite any events. Several events can be associated with one aggregate ID.
+     * <p>NOTE: does not rewrite any events. Several events can be associated with one
```

Comment:
Let's settle down on whether we use `<b>` around "NOTE" or the capitalization is good enough even with no boldening. See the similar case above.